std-bits/table/src/main/java/org/enso/table/data/column/storage/StringStorage.java

```diff
@@ -51,8 +52,8 @@ protected Storage runVectorizedZip(String name, Storage argument) {
   }
 
   @Override
-  public Storage fillMissing(Object arg) {
-    if (arg instanceof String) {
+  public Storage fillMissing(Value arg) {
+    if (arg.isString()) {
       return fillMissingHelper(arg, new StringBuilder(size()));
```

Comment:
Do we need `arg.asString()`?

Reply:
No, we even cannot - because `fillMissingHelper` is taking a Value and `String </: Value`. `fillMissingHelper` will handle the correct conversion on its own, this check is only done to initialize the correct builder.